third_party/3/packaging/markers.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Dict, Optional
+
+class InvalidMarker(ValueError): ...
+class UndefinedComparison(ValueError): ...
+class UndefinedEnvironmentName(ValueError): ...
+
+class Node:
+    value: Any = ...
+    def __init__(self, value: Any) -> None: ...
+    def serialize(self) -> str: ...
+
+class Variable(Node):
+    def serialize(self) -> str: ...
+
+class Value(Node):
+    def serialize(self) -> str: ...
+
+class Op(Node):
+    def serialize(self) -> str: ...
+
```

Comment:
There's a number of other objects like `VERSION_CMP` at runtime. They're not explicitly private, but I don't know if they're part of the intended API of the package. They're not in `__all__` but neither are Variable, Value, and Op.

requirements.py has a similar set of all-caps pyparsing-based names. They look like implementation details to me but maybe we'll have to add them to the stub at some point. For now I'm OK with not including them.